src/_utils.ts

```diff
@@ -57,3 +60,16 @@ export function isPlainObject(value: unknown): boolean {
 
   return true;
 }
+
+export function adapterUtils(peers: Set<Peer>) {
+  return {
+    peers,
+    publish(topic: string, message: any, options) {
+      const firstPeer = peers.values().next().value as Peer;
+      if (firstPeer) {
+        firstPeer.send(message, options);
```

Comment:
Hi @pi0, thank you for this feature 🥇 
I was about to use it although I think there is a bug here. You are publishing to `firstPeer` without checking whether it is subscribed to a topic.

I don't think there is currently an easy way to see which topics a peer is subscribed to without checking the protected property `_topics`, but simple fix for me was to just do:

```ts
if (firstPeer['_topics'].has(topic)) firstPeer.send(message, options);
```

BTW. I wish Nitro would expose these utils somewhere (I couldn't find any way to access them from Nuxt)